### PR TITLE
MINOR: Fix typo in javadoc of `flatMapValues`

### PR DESCRIPTION
**Pull request by miguno**

### CODE REVIEW

**Comment by miguno** [comment]
Ping any of @guozhangwang @mjsax @dguy @enothereska

**Review by dguy — APPROVED** [review]
LGTM

File: streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java [hunk 1]
```diff
@@ -238,7 +238,7 @@
      * The example below splits input records {@code <null:String>} containing sentences as values into their words.
      * <pre>{@code
      * KStream<byte[], String> inputStream = builder.stream("topic");
-     * KStream<Integer, String> outputStream = inputStream.flatMap(new ValueMapper<String, Iterable<String>> {
+     * KStream<Integer, String> outputStream = inputStream.flatMapValues(new ValueMapper<String, Iterable<String>> {
```
Comment (mjsax) [inline streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java]: `KStream<Integer, String> outputStream` should be `KStream<byte[], String>` ?

**Review by mjsax — CHANGES_REQUESTED** [review]

File: streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java (hunk 1 above)
Comment (miguno) [reply streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java]: @mjsax Correct -- and fixed.  If only I could remember who wrote the original javadoc. ;-)

Comment (mjsax) [reply streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java]: If I could only remember, who not reviewed the original JavaDoc PR :P

**Review by mjsax — APPROVED** [review]
LGTM

**Review by guozhangwang — APPROVED** [review]
Merged to trunk.

**Pull request closed without merging**